ui/src/app/job-list/table/table.component.spec.ts

```diff
@@ -265,6 +292,21 @@ describe('JobsTableComponent', () => {
     expect(isGroupAbortIsEnabled()).toBeTruthy();
   }))
 
+  it('disables the bulk edit button when there is a bulkEditable field but less than two jobs selected', async(() => {
```

Comment:
I don't think I agree with this >=2 behavior. I don't see any advantage to disabling bulk edits for a single job. Agreed, they could do it more quickly by just selecting the specific job directly, but maybe they haven't discovered that particular interaction or they don't want to think about how many they selected.